server/sandbox.go

```diff
@@ -599,3 +618,18 @@ func getSELinuxLabels(selinuxOptions *pb.SELinuxOption) (processLabel string, mo
 	}
 	return label.InitLabels(label.DupSecOpt(processLabel))
 }
+
+// sysctlsFromPodAnnotation parse annotation in form of comma separated list of sysctl_name=value key-value pairs, returns a list of {Name, Value} struct.
+func sysctlsFromPodAnnotation(annotation string) ([]sysctl, error) {
+	kvs := strings.Split(annotation, ",")
+	sysctls := make([]sysctl, len(kvs))
+	for i, kv := range kvs {
+		cs := strings.Split(kv, "=")
+		if len(cs) != 2 || len(cs[0]) == 0 {
```

Comment:
@Crazykev I don't know if the value can be empty, hence I asked, maybe @mrunalp knows more